testkit/tests/system_api.rs

```diff
@@ -52,3 +52,14 @@ fn test_network() {
     assert_eq!(info.protocol_version, PROTOCOL_MAJOR_VERSION);
     assert!(info.services.is_empty());
 }
+
+#[test]
+fn test_consensus_status_false() {
```

Comment:
`test_consensus_status_false` checks that `consensus_status` is true. Seems legit :upside_down_face:

Reply:
The same situation with test_healthcheck_connectivity_false.

Reply:
If I don't miss anything, tests look like the following:
```
#[test]
fn test_healthcheck_connectivity_false() {
    let testkit = TestKitBuilder::validator().with_validators(2).create();
    let api = testkit.api();
    let info: HealthCheckInfo = api.get(ApiKind::System, "v1/healthcheck");
    let expected = HealthCheckInfo {
        connectivity: false,
    };
    assert_eq!(info, expected);
}
```

```
#[test]
fn test_consensus_status_false() {
    let testkit = TestKitBuilder::validator().with_validators(1).create();
    let api = testkit.api();
    let info: ConsensusStatusInfo = api.get(ApiKind::System, "v1/consensus_status");
    let expected = ConsensusStatusInfo {
        status: true,
    };
    assert_eq!(info, expected);
}
```

`test_healthcheck_connectivity_false` checks that the `connectivity` field is false (which is perfectly consistent with the name).

`test_consensus_status_false` checks that the `status` field is true (which is kind of confusing).

Or maybe I miss something?

Reply:
healthcheck  with 2 validators must return connectivity = true,  but it failed. It seems some trouble with testkit, or testkit isn't suitable to perform these tests.

Reply:
Just to be clear: the only confusing thing is the name of the test which tells us that we are checking if status is false, and check for status to be true in the body.